docs/website/root/manual/getting-started/run-signer-node.md

```diff
@@ -113,6 +113,30 @@ Note that this guide works on a Linux machine only.
 
 :::
 
+:::info
+
+Compare the version of your Cardano node with the minimum supported versions listed in the [`networks.json`](https://github.com/input-output-hk/mithril/blob/main/networks.json) to verify its compatibility with the Mithril signer.
+
+First, check the version of your Cardano node by running the following command:
+
+```bash
+cardano-node --version
+```
+
+Then, refer to the minimum supported versions listed in the the [`networks.json`](https://github.com/input-output-hk/mithril/blob/main/networks.json) file.
+
+You can also fetch the minimum supported version for your network using the command below:
+```bash
+wget -q -O - https://raw.githubusercontent.com/input-output-hk/mithril/main/networks.json | jq -r '."**YOUR_CARDANO_NETWORK**"."cardano-minimum-version"."mithril-signer"'
```

Comment:
Do we need the `"` in the `jq` args here and below?

Reply:
Yes, `jq` doesn't handle the hyphen correctly.
However, I can remove the `"` around the **YOUR_CARDANO_NETWORK** as the Cardano networks are currently written in a single word.